src/modules/sse/SseModule.ts

```diff
@@ -88,11 +89,12 @@ export default class SseModule extends ConnectorRuntimeModule<SseModuleConfigura
 
         const services = this.runtime.getServices();
 
-        const syncResult = await services.transportServices.account.syncEverything();
-        if (syncResult.isError) {
-            this.logger.error(syncResult);
-            return;
-        }
+        await correlationIdLibrary.withId(async () => {
+            const syncResult = await services.transportServices.account.syncEverything();
+            if (syncResult.isError) {
+                this.logger.error(syncResult);
+            }
+        });
```

Comment:
maybe this should happen in all the UseCases of the runtime to not have to think about to do it here?

Reply:
I think this would affect https://github.com/nmshd/runtime/pull/268 so please wait to merge this until this discussion is resolved.

Reply:
The problem with doing it in the runtime is that the error log from the connector would not have that correlation ID as it is in the parent scope.

Reply:
But in general we could add it in the generic execute handler of the runtime and check if there is already one set and reuse this ID

Reply:
Hmm, that's bad. But we could do it ALSO in the runtime, right? Then if it's defined it would pick up the ID defined here, and if not create one.